make image-build-uds-tokenizer failed

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!

-->

**What happened**:

Clone the code and make image-build-uds-tokenizer

```console
$ make image-build-uds-tokenizer
  go: downloading go1.25.7 (darwin/arm64)
  Container tool 'docker' found.
  ==== Building UDS Tokenizer image ghcr.io/llm-d/llm-d-uds-tokenizer:dev ====
  kv-cache module not found, downloading Go modules...
  docker build \
          --platform linux/arm64 \
          -t ghcr.io/llm-d/llm-d-uds-tokenizer:dev \
          -f /services/uds_tokenizer/Dockerfile \
          /services/uds_tokenizer
  [+] Building 0.0s (0/0)                                                    docker:desktop-linux
  ERROR: failed to build: unable to prepare context: path "/services/uds_tokenizer" not found
  make: *** [image-build-uds-tokenizer] Error 1
```

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- llm-d-scheduler version (use `git describe --tags --dirty --always` if you built from source, or specify the tag if you used a tagged version or image):
- Cloud provider or hardware configuration:
- Install tools:
- Others:

--- a/Makefile
+++ b/Makefile
@@ -190,12 +190,12 @@ image-build-uds-tokenizer: check-container-tool ## Build UDS tokenizer image fro
 	if [ -z "$$KV_CACHE_PATH_CHECK" ]; then \
 		echo "Error: Could not find kv-cache module even after download."; \
 		exit 1; \
-	fi
+	fi; \
 	$(CONTAINER_RUNTIME) build \
 		--platform linux/$(TARGETARCH) \
 		-t $(UDS_TOKENIZER_IMAGE) \
-		-f $(KV_CACHE_PATH)/services/uds_tokenizer/Dockerfile \
-		$(KV_CACHE_PATH)/services/uds_tokenizer
+		-f $$KV_CACHE_PATH_CHECK/services/uds_tokenizer/Dockerfile \
+		$$KV_CACHE_PATH_CHECK/services/uds_tokenizer
 
 .PHONY: image-build-%
 image-build-%: check-container-tool ## Build Container image using $(CONTAINER_RUNTIME)

